skbio/stats/tests/test_composition.py

```diff
@@ -546,12 +546,197 @@ def test_ancom_basic_counts(self):
         assert_data_frame_almost_equal(original_table, test_table)
         # Test to make sure that the input table hasn't be altered
         pdt.assert_series_equal(original_cats, test_cats)
-        exp = pd.DataFrame({'W': np.array([5, 5, 2, 2, 2, 2, 2]),
-                            'reject': np.array([True, True, False, False,
-                                                False, False, False],
-                                               dtype=bool)})
+        exp = pd.DataFrame(
+            {'W': np.array([5, 5, 2, 2, 2, 2, 2]),
+             'Reject null hypothesis': np.array([True, True, False, False,
+                                                 False, False, False],
+                                                dtype=bool)})
+        assert_data_frame_almost_equal(result[0], exp)
+
+    def test_ancom_basic_counts_shuffled_order(self):
```

Comment:
This test basically exists as `test_ancom_basic_counts_swapped`. Instead of testing the first data frame (which `test_ancom_basic_counts_swapped` is testing), can you change this test to assert the second data frame is correct?